amr-wind/equation_systems/tke/source_terms/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 target_sources(${amr_wind_lib_name} PRIVATE
   KsgsM84Src.cpp
   KwSSTSrc.cpp
+  Krans.cpp
```

Comment:
Agreed with @sayerhs. Lets have a more specific name for this.

Reply:
Let me finish the other changes and before final commit, change the name.

Reply:
Names changed.